lib/order.rb

```diff
@@ -0,0 +1,101 @@
+require 'csv'
+require_relative 'customer'
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products # provided a hash with key prod name, value price)
+    @customer = customer # pass in from customer class
+    @fulfillment_status = fulfillment_status
+
+    raise ArgumentError.new("Invalid fulfillment status") if [:pending, :paid, :processing, :shipped, :complete].include?(@fulfillment_status) != true
```

Comment:
`include?` returns `true` or `false` already so you can simply treat it as a condition.  

Also, it's a good idea to include the invalid argument in the message for the `ArgumentError`:
```suggestion
    raise ArgumentError.new("Invalid fulfillment status: #{@fulfillment_status}") unless [:pending, :paid, :processing, :shipped, :complete].include?(@fulfillment_status)
```